doc/manuals/devel/httpHeaders.md

```diff
@@ -0,0 +1,329 @@
+When `-multiservice` is used Orion includes the `Fiware-Service` header in the notifyContextRequest and notifyContextAvailability request messages associated to subscriptions in the given tenant/service (except for the default service/tenant, in which case the header is not present).
+		
+    POST http://127.0.0.1:9977/notify    
+    Content-Length: 725    
+    User-Agent: orion/0.13.0    
+    Host: 127.0.0.1:9977      
+    Accept: application/json    
+    Fiware-Service: t_02    
+    Content-Type: application/json    
+    {    
+	   ...    
+    } 
+      
+ Regarding service/tenant name syntax, it must be a string of alphanumeric characters (and the `_` symbol). 
+ Maximum length is 50 characters, which should be enough for most use cases. 
+ Orion Context Broker interprets the tenant name in lowercase, thus, although you can use tenants such as in updateContext `MyService` 
+ it is not advisable, as the notifications related with that tenant will be sent with `myservice` and, 	in that sense, 
+ it is not coherent the tenant you used in updateContext compared with the one that Orion sends in notifyContextRequest.
+	
+[Top](#top)
+
+
+## 20. Fiware-Servicepath
+
+Fiware-ServicePath is an optional header used in requests.
+Fiware-ServicePath header is included in notification requests sent by Orion.
+When `-multiservice` is used, Orion uses the `Fiware-Service` HTTP header in the request to identify the service/tenant.
+If the header is not present in the HTTP request, the default service/tenant is used.	
+The scope to use is specified using the Fiware-ServicePath HTTP in update/query request. 
+
+    Fiware-ServicePath: /Madrid/Gardens/ParqueNorte/Parterre1
+		
+Queries without Fiware-ServicePath resolve to `/#`.
+	
+It is assumed that all the entities created without Fiware-ServicePath (or that don't include service path information in the database) 
+belongs to a root scope `/` implicitely. All the queries without using Fiware-ServicePath (including subscriptions) are on `/#` implicitly. 
+
+While entities belong to services and servicepaths, subscriptions and registrations belong only to the service. 
+The servicepath in subscriptions and registrations doesn't denote sense of belonging, but is the expression of the 
+query associated to the subscription or registration.
+
+Fiware-ServicePath header is ignored in `GET /v2/subscriptions/{id}` and `GET /v2/registrations/{id}` operations, 
+as the id fully qualifies the subscription or registration to retrieve.
+
+Fiware-ServicePath header is taken into account in `GET /v2/subscriptions` and `GET /v2/registrations` in order to narrow 
+down the results to subscriptions/registrations that use exactly that service path as query.
+
+At the present moment hierarchical service paths (i.e. the ones using ending with #) are not allowed in registrations.
+Fiware-Service and Fiware-ServicePath are set at entity creation time using HTTP headers in the entity creation REST request.
```

Comment:
Again, instead of duplicanting text, I think it would be better to include a reference to the document describing service path in the users manual.

Reply:
Fixed in a427bd9